contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributions
+
+Contributions are always welcome :smile ! you can contribute in many ways. 
+
+## Reporting Bugs
+
+Report Bugs at https://github.com/sforaidl/jeta/issues
+
+When reporting bugs please 
+- Include the version and some information about your local environment setup. 
+- Way to reproduce the bug
+
+## Fixing Bugs: 
+
+Issues tagged with `help wanted`,`good first issues` and `bug` are open for everyone.
+
+## Get Started
+
+Follow the steps to setup jeta for local development process
+
+1) Fork the Jeta repo on github
+
+2) clone your fork on local machine using following terminal command. 
+```bash
+git clone https://github.com/<insert your github username>/jeta.git
+```
+
+3) Create a conda virtual environment using `environment.yml` file. 
```

Comment:
We (currently use a `requirements.txt` file (since it goes with pip easily). I prefer using `pip` since it's simpler.
However, we can also add a `environment.yml` file

Reply:
Yeah, having environment.yml and requirements.txt (both serve the same purpose) is redundant.  I think it's better to remove it. Will install everything through pip